src/contents/gitpod-dashboard.ts

```diff
@@ -0,0 +1,24 @@
+import type { PlasmoCSConfig } from "plasmo";
+import { Storage } from "@plasmohq/storage"
+import { isSiteGitpod } from "../button/button-contributions";
+import { STORAGE_KEY_ADDRESS } from "~storage";
+
+export const config: PlasmoCSConfig = {
+  matches: ["https://*/*"]
+}
+
+if (isSiteGitpod()) {
+  sessionStorage.setItem("browser-extension-installed", "true");
+  (async () => {
+    const storage = new Storage();
+    const gitpodEndpoint = await storage.getItem(STORAGE_KEY_ADDRESS);
```

Comment:
We don't need to distinguish whether the address points to this gitpod installation, because we tell users to install the extension, which is confusing if you have it installed.
 
We should instead show an overlay that allows users to set the address, but I believe we can do that in an overlay without interaction with gitpod dashboard.

Reply:
We can keep this separate and do it purely in-extension 👍

Reply:
Adressed in https://github.com/gitpod-io/browser-extension/pull/92/commits/e63d3f243f9183f8521ef2c046be709f46f015d0